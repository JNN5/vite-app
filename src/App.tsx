import { RouterProvider } from "react-router-dom";
import { router } from "./app/router";

export default function App() {
  return (
    <div className="App">
      <RouterProvider router={router} />
    </div>
  );
}
