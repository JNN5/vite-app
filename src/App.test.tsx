import { screen } from "@testing-library/react";
import { render } from "./test-setup/test-utils";

import App from "./App";

describe("Page", () => {
  it("renders page", () => {
    render(<App />);

    screen.debug();
  });

  it("matches screenshot", () => {
    const { asFragment } = render(<App />);

    expect(asFragment()).toMatchSnapshot();
  });
});
