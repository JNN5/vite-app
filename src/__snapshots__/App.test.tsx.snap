// Vitest Snapshot v1

exports[`Page > matches screenshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  />
</DocumentFragment>
`;
